doc/path.md

```diff
@@ -13,13 +13,22 @@ For instance, a "then" branch in an if/then/else is a role (and not a node). All
 
 ### Evaluating AST paths
 
-Paths are used to find code elements, from a given root element.
+Paths are used to find code elements, from a given root element(e.g. `model.getRootPackage()`).
```

Comment:
```suggestion
Paths are used to find code elements, from a given root element.
```
I would not suggest `CtModel#getRootPackage` here, as that does not play nice with modules. `CtModule#getRootPackage` or something else would be better.